src/Cli/dotnet/commands/dotnet-workload/install/NetSdkMsiInstallerClient.cs

```diff
@@ -314,19 +314,16 @@ public void EradicateAllWorkloadInstallationRecords(SdkFeatureBand currentFeatur
             {
                 Log?.LogMessage($"Detected band with installation record: '{potentialBandToClean}'.");
 
-                if (potentialBandToClean.CompareTo(currentFeatureBand) < 1)
-                {
-                    Log?.LogMessage($"'{potentialBandToClean}' is confirmed to be less than or current to the current band.");
-
-                    var workloadInstallationRecordIds = RecordRepository.GetInstalledWorkloads(potentialBandToClean);
-                    foreach (WorkloadId workloadInstallationRecordId in workloadInstallationRecordIds)
-                    {
-                        Log?.LogMessage($"Workload {workloadInstallationRecordId} for '{potentialBandToClean}' has been marked for deletion.");
-                        RecordRepository.DeleteWorkloadInstallationRecord(workloadInstallationRecordId, potentialBandToClean);
-                    }
+                Log?.LogMessage($"'{potentialBandToClean}' is confirmed to be less than or current to the current band.");
```

Comment:
Should that be `less than or equal`?

Reply:
Good point. I should've removed this line.